README.md

```diff
@@ -1 +1,79 @@
-# taskratchet-mobile
```

Comment:
Should we keep this as the top-level title for the readme?

Reply:
Updated